src/maintainer/adding_pkgs.rst

```diff
@@ -3,6 +3,13 @@
 Contributing packages
 *********************
 
+
+A maintainer is an individual who is responsible for maintaining and updating one or more feedstock repositories and packages as well as their future versions. They have push access to the feedstock repositories of the packages they maintain and can merge `PR <https://conda-forge.org/docs/misc/00_intro.html#glossary>`__ into it. See `Maintainers Role. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__
```

Comment:
just to be more specific it should be good to point out that they have push rights only for the packages they maintain.

Reply:
That is, I should write : They have push access to the feedstock repositories of only the packages they maintain?

Reply:
I think we can follow the Governance guidelines here:

Reply:
They have push access to the feedstock repositories of only the packages they maintain and can merge `PR <https://conda-forge.org/docs/misc/00_intro.html#glossary>`__ into it.

I think this sentence is alright and complies with what you suggested. What do you say?